static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -98,11 +98,23 @@
         }
     }
 
+    .content__head--desktop {
+        height: 100vh;
+        max-height: 800px;
+    }
+
+    .content__header {
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        right: 0;
+    }
+
     @include mq($until: desktop) {
         .content__head--mobile {
             position: relative;
-            height: 100vh;
-            max-height: 800px;
+            display: table-cell;
+            vertical-align: bottom;
```

Comment:
Do we need the parent to have display:table for this to work everywhere, or is this enough?